index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css"
+      integrity="sha384-b6lVK+yci+bfDmaY1u0zE8YYJt0TZxLEAFyYSLHId4xoVvsrQu3INevFKo+Xir8e"
+      crossorigin="anonymous"
+      />
+      <title>My portfolio</title>
+  </head>
+  <body>
+      <!-- <header>
+        <nav class="nav bar">
+          <a href="#">Risk</a>
+          <i class="bi bi-list"></i>
+        </nav>
+      </header> -->
+      <section class="parent">
+        <nav>
+          <div class="nav-bar">
+            <div>
+              RISK
+            </div>
+            <div>
+              <i class="bi bi-list"></i>
+            </div>
+          </div>
+        </nav>
+        <section>
+          <section class="bg-img">
+              <div>
+                <h1 class="name">I'm Maris<br/>Glad to see</br> you!</h1>
+                <p>
+                  I’m a software developer! I can help you<br>
+                  build a product , feature or
+                  website Look<br>
+                  through some of my work and<br>
+                  experience! If you like what
+                  you see and<br>
+                  have a project you need coded, don’t<br>
+                  hesitate to contact me.
+                </p>
+              </div>
+              <div>
+                <h3 class="connect">LET'S CONNECT</h3>
+              </div>
+
+              <div>
+                <a href=""><img src="images/social media icons.png" alt="Pictures of Media Icons"></a>
```

Comment:
- [ ] In my opinion, it would be advantageous to utilize separate social media links, allowing you to incorporate your own social media links into the website. Consider the following code snippet as a suggestion:

```suggestion
           <ul>
      <li>link-1</li>
      <li>link-2</li>
      <li>link-2</li>
      <li>link-3</li>
      <li>link-3</li>
    </ul>
```

- [ ] By using separate social media links, you can easily customize and add your desired social media links to the website.